custom_components/kia_uvo/__init__.py

```diff
@@ -80,9 +80,9 @@ async def update(event_time):
         await vehicle.update()
         if (event_time.hour < NO_FORCE_SCAN_HOUR_START and event_time.hour >= NO_FORCE_SCAN_HOUR_FINISH):
             _LOGGER.debug(f"{DOMAIN} - We are in force hour zone {event_time}")
-            _LOGGER.debug(f"{DOMAIN} - Check last update of vehicle {vehicle.last_updated} {datetime.now()} {FORCE_SCAN_INTERVAL} {datetime.now() - vehicle.last_updated > FORCE_SCAN_INTERVAL}")
+            _LOGGER.debug(f"{DOMAIN} - Check last update of vehicle {vehicle.last_updated} {datetime.now(timezone.utc)} {FORCE_SCAN_INTERVAL} {datetime.now(timezone.utc) - vehicle.last_updated > FORCE_SCAN_INTERVAL}")
```

Comment:
this might be simpler `datetime.utcnow()`

Reply:
that gives a native datetime rather than an aware one - I had to double check this too, so doesn't do the subtraction.